firefox-ios/Client/Frontend/Settings/RemoteSettingsOptionViewController.swift

```diff
@@ -0,0 +1,141 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import UIKit
+
+class RemoteSettingsOptionViewController: UIViewController, UITableViewDelegate, UITableViewDataSource {
```

Comment:
You can ignore this for now as this is built for debugging but you should be able to check it out via the debug menu